src/igvfd/types/sample.py

```diff
@@ -37,6 +37,9 @@ class Biosample(Sample):
     item_type = 'biosample'
     base_types = ['Biosample'] + Sample.base_types
     schema = load_schema('igvfd:schemas/biosample.json')
+    audit_inherit = [
+        'sample'
+    ]
```

Comment:
I don't think we discussed audit_inheritance, why is it added here?

Reply:
This is removed.